sig-contributor-experience/annual-report-2023.md

```diff
@@ -3,17 +3,68 @@
+        - [Onboarded SIG Etcd to Kubernetes GitHub Organisation & Prow](https://github.com/kubernetes/community/pull/7372) (major thanks to @cblecker, @mrbobbytables)
+        - [Peribolos tool, extended to manage repo level permissions under GitHub teams](https://github.com/kubernetes/org/pull/2614)
+
+    - Events:
+        - 3 successful Kubernetes contributors summits:
+            - [Europe, hosted in Amsterdam, Netherlands](https://www.kubernetes.dev/events/2023/kcseu/)
+            - [China, hosted in Shanghai](https://www.kubernetes.dev/events/2023/kcscn/) (special thanks to @pacoxu for [leading the event](https://github.com/kubernetes/community/issues/7510))
+            - [North America, hosted in Chicago, Illinois](https://www.kubernetes.dev/events/2023/kcsna/)
+
+    - Mentoring:
+        - 1 successful mentoring cohort:
+            - [Kustomize maintainer training cohort](https://groups.google.com/a/kubernetes.io/g/dev/c/M5OphEVsv5o/m/zc6G4H15AAAJ)
+
+    - Elections:
+        - [Kubernetes Steering Committee Elections, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/bZpmcKA_900/m/EnlPmiDBAQAJ)
+        - Facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
+
+    - Contributor-comms:
+        - [Subproject name change](https://github.com/kubernetes/community/commit/3613c1d2b33ab2283dc28f0215ff772668add39c): "Marketing Council" (earlier) -> "Contributor Communications" (Contributor Comms for short)
+        - Got access to [Kubernetes Linkedin account](https://www.linkedin.com/company/kubernetes/)
+        - Got access to Kubernetes X (earlier, Twitter) account – [@kubernetesio](https://x.com/kubernetesio)
+        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
+        - 5 SIG spotlight blogs (2/5 contributed by new contributors)
+            - [Spotlight on SIG CLI](https://kubernetes.io/blog/2023/07/20/sig-cli-spotlight-2023/)
+            - [Spotlight on SIG Testing](https://kubernetes.io/blog/2023/11/24/sig-testing-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Conformance](https://kubernetes.io/blog/2023/10/05/sig-architecture-conformance-spotlight-2023/)
+            - [Spotlight on SIG Architecture: Production Readiness](https://kubernetes.io/blog/2023/11/02/sig-architecture-production-readiness-spotlight-2023/)
+            - [Introducing SIG etcd](https://kubernetes.io/blog/2023/11/07/introducing-sig-etcd/)
+        - New leadership roles in subproject:
+            - Onboarded 1 [Comms Tech Lead](https://github.com/kubernetes/community/commit/ba3d34a001abd3d50004f87ae44385efa5b2695d)
+            - Onboarded 1 [Social Media Coordinator shadow](https://github.com/kubernetes/community/pull/7438)
```

Comment:
@kaslin, I've added rest of the contributor-comms points (we discussed during yesterday's (June 5, 2024) leads sync call). Please review. Thanks!